examples/storage/cassandra/README.md

```diff
@@ -215,6 +215,10 @@ spec:
           capabilities:
             add:
               - IPC_LOCK
+        lifecycle:
```

Comment:
@rootfs that change was introduced when i run `make update` after a rebase.  Not sure which script did it.